### PR TITLE
fixed Scope and SentryEvent attributes

### PR DESCRIPTION
**Pull request by marandaneto**
removed not used attributes from Scope and removed the inheritance from SentryEvent and duplicated needed events

### CODE REVIEW

File: sentry-core/src/main/java/io/sentry/Scope.java [hunk 1]
```diff
@@ -1,24 +1,20 @@
...
   private String transaction;
-  private String environment;
   private User user;
-  private Request request;
-  private SdkVersion sdkVersion;
   private List<String> fingerprint = new ArrayList<>();
-  private CopyOnWriteArrayList<Breadcrumb> breadcrumbs = new CopyOnWriteArrayList<>();
-  private ConcurrentHashMap<String, String> tags = new ConcurrentHashMap<>();
-  private ConcurrentHashMap<String, Object> extra = new ConcurrentHashMap<>();
+  private List<Breadcrumb> breadcrumbs = new CopyOnWriteArrayList<>();
```
Comment (bruno-garcia) [inline sentry-core/src/main/java/io/sentry/Scope.java]: For the private field I'd leave the actual type explicit here.

File: sentry-core/src/main/java/io/sentry/Scope.java [hunk 2]
```diff
@@ -52,22 +40,6 @@ public void setUser(User user) {
     this.user = user;
   }
 
-  public Request getRequest() {
```
Comment (bruno-garcia) [inline sentry-core/src/main/java/io/sentry/Scope.java]: Why did Scope lose request?

File: sentry-core/src/main/java/io/sentry/SentryEvent.java [hunk 3]
```diff
@@ -19,6 +26,16 @@
...
   private SentryValues<SentryThread> threads;
   private SentryValues<SentryException> exceptions;
+  private SentryLevel level;
+  private String transaction;
+  private String environment;
+  private User user;
+  private Request request;
+  private SdkVersion sdkVersion;
+  private List<String> fingerprint = new ArrayList<>();
+  private List<Breadcrumb> breadcrumbs = new CopyOnWriteArrayList<>();
```
Comment (bruno-garcia) [inline sentry-core/src/main/java/io/sentry/SentryEvent.java]: Event doesn't need `CopyOnWrite`. It is not thread-safe so we can use simple(cheaper) types here.

File: sentry-core/src/main/java/io/sentry/protocol/SdkVersion.java [hunk 4]
```diff
@@ -1,13 +1,12 @@
...
 
-import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.CopyOnWriteArrayList;
 
 public class SdkVersion {
   private String name;
   private String version;
-  private CopyOnWriteArrayList<Package> packages = new CopyOnWriteArrayList<>();
+  private List<Package> packages = new CopyOnWriteArrayList<>();
```
Comment (bruno-garcia) [inline sentry-core/src/main/java/io/sentry/protocol/SdkVersion.java]: What's the motivation of using the abstract type here?

File: sentry-core/src/main/java/io/sentry/Scope.java (hunk 2 above)
Comment (marandaneto) [reply sentry-core/src/main/java/io/sentry/Scope.java]: @HazAT said 

> and remove environment, request, sdkVersion from the scope

**Pull request merged**